### DISCUSSION TITLE
APImanager is not running in GCP

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-Ed/Community#270] [Installation] [answered]
Hi team,

CSP-Googlecloud

APImanager is not running in kubernetes.

`prasath_sivasubramanian@jenkins:~$ kubectl describe po/apimanager-687d5cf4f6-f2wvv -n dev
Name:         apimanager-687d5cf4f6-f2wvv
Namespace:    dev
Priority:     0
Node:         gke-gke-dev-cluster-default-pool-586d4848-8dxk/10.0.1.9
Start Time:   Tue, 29 Nov 2022 23:48:58 +0530
Labels:       app=apimanager
              pod-template-hash=687d5cf4f6
Annotations:  fluentbit.io/parser: kong
Status:       Running
IP:           192.168.0.18
IPs:
  IP:           192.168.0.18
Controlled By:  ReplicaSet/apimanager-687d5cf4f6
Containers:
  apimanager:
    Container ID:   containerd://f85d81d4132680660a9de53ecb87e87108b8e1644d5f1d561cb8ad291f814cd1
    Image:          asia.gcr.io/upsmf-368011/kong:0.14.1
    Image ID:       asia.gcr.io/upsmf-368011/kong@sha256:bb423798525abd17cdd1d1691e9de3870d9befa9076b7ba7734cd51b5064308c
    Ports:          8000/TCP, 8001/TCP
    Host Ports:     0/TCP, 0/TCP
    State:          Running
      Started:      Tue, 29 Nov 2022 23:49:01 +0530
    Ready:          False
    Restart Count:  0
    Limits:
      cpu:     1
      memory:  1Gi
    Requests:
      cpu:      100m
      memory:   100Mi
    Liveness:   http-get http://:8001/status delay=15s timeout=5s period=15s #success=1 #failure=2
    Readiness:  http-get http://:8001/status delay=15s timeout=5s period=15s #success=1 #failure=2
    Environment Variables from:
      apimanager-config  ConfigMap  Optional: false
    Environment:         <none>
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-5tdjx (ro)
Conditions:
  Type              Status
  Initialized       True 
  Ready             False 
  ContainersReady   False 
  PodScheduled      True 
Volumes:
  kube-api-access-5tdjx:
    Type:                    Projected (a volume that contains injected data from multiple sources)
    TokenExpirationSeconds:  3607
    ConfigMapName:           kube-root-ca.crt
    ConfigMapOptional:       <nil>
    DownwardAPI:             true
QoS Class:                   Burstable
Node-Selectors:              <none>
Tolerations:                 node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
                             node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
Events:
  Type     Reason     Age   From               Message
  ----     ------     ----  ----               -------
  Normal   Scheduled  32s   default-scheduler  Successfully assigned dev/apimanager-687d5cf4f6-f2wvv to gke-gke-dev-cluster-default-pool-586d4848-8dxk
  Normal   Pulling    31s   kubelet            Pulling image "asia.gcr.io/upsmf-368011/kong:0.14.1"
  Normal   Pulled     29s   kubelet            Successfully pulled image "asia.gcr.io/upsmf-368011/kong:0.14.1" in 1.830893203s
  Normal   Created    29s   kubelet            Created container apimanager
  Normal   Started    29s   kubelet            Started container apimanager
  Warning  Unhealthy  1s    kubelet            Liveness probe failed: Get "http://192.168.0.18:8001/status": dial tcp 192.168.0.18:8001: connect: connection refused
  Warning  Unhealthy  1s    kubelet            Readiness probe failed: Get "http://192.168.0.18:8001/status": dial tcp 192.168.0.18:8001: connect: connection refused`

**Comment by Krishnaj20** [comment, marked as answer]
@keshavprasadms @gandham-santhosh - FYA please

**Reply by tsprasath** [reply]
I have done some ground. it works for me. Please mark it as resolved.
